assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -40,8 +40,18 @@ const VERTICAL_HANDLES = ['n', 's'];
 const HORIZONTAL_HANDLES = ['e', 'w'];
 const DIAGONAL_HANDLES = ['nw', 'ne', 'sw', 'se'];
 
-function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
-  const moveable = useRef(null);
+function SingleSelectionMovable({
+  selectedElement,
+  targetEl,
+  pushEvent,
+  isEditMode,
+  editMoveableRef,
+  setActionHappening,
+}) {
+  let moveable = useRef(null);
+  if (isEditMode) {
+    moveable = editMoveableRef;
```

Comment:
Here we're using the ref that was passed from the edit mode -- so that we could update the Moveable if the height has changed in the edit mode without the element properties being updated yet.

Reply:
I think we had combinedRefs code somewhere?